CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.29.0] - 2023-01-16
```

Comment:
Should this be '2024-01-16' ?

Reply:
@saadberry I kept them same because I am planning to deploy these 2 PRs as a single release.

Reply:
Alright, thanks